system/Session/Handlers/FileHandler.php

```diff
@@ -48,13 +48,6 @@ class FileHandler extends BaseHandler
      */
     protected $fileNew;
 
-    /**
-     * Whether IP addresses should be matched.
-     *
-     * @var bool
-     */
-    protected $matchIP = false;
```

Comment:
Why is this removed? Is this duplicated by BaseHandler?

Reply:
Yes it is